src/controllers/signup.js

```diff
@@ -0,0 +1,52 @@
+import express from 'express';
+import bcrypt from 'bcryptjs';
+import signupvalidation from '../validations/signup';
+import { SignupModel } from '../database/models';
+
+const router = express.Router();
+
+// Register User
+
+router.post('/signup', async (req, res) => {
```

Comment:
May it be good for us if you separate routes from controllers

Reply:
It's done